saleor/graphql/shop/types.py

```diff
@@ -53,7 +62,10 @@ class Shop(graphene.ObjectType):
         Geolocalization,
         description='Customer\'s geolocalization data.')
     authorization_keys = graphene.List(
-        AuthorizationKey, description='List of configured authorization keys.',
+        AuthorizationKey,
+        description='''List of configured authorization keys. Authorization
+        keys are used to enable thrid party OAuth authorization (currently
```

Comment:
To be precise it's `OAuth2`

Reply:
Is the concept of `client_id` and `client_secret` fields specific only to OAuth2 and not valid in OAuth1? I meant general concept of OAuth.

Reply:
As far as I know it's the same concept, matter of fact if we want to say precisely that OAuth2 authorization is supported by us, or that this feature can be used for OAuth in general

Reply:
`python-social-auth` which we use under the hood provides backends for both OAuth1 and OAuth2 authorizations. All of them could be potentially enabled in Saleor, so I think that generally saying about OAuth here should be fine.